tests/sources/test_confluence.py

```diff
@@ -650,8 +677,8 @@ async def test_fetch_spaces():
         )
 
         with mock.patch("aiohttp.ClientSession.get", return_value=async_response):
-            async for response, _, _ in source.fetch_spaces():
-                assert response == EXPECTED_SPACE
+            async for response in source.confluence_client.fetch_spaces():
+                assert response["id"] == EXPECTED_SPACE["_id"]
```

Comment:
This is a much weaker assertion. You should be able to still validate that the source's output is the same as it was before.